cost-analyzer/templates/cost-analyzer-deployment-template.yaml

```diff
@@ -1223,6 +1158,18 @@ spec:
           {{- toYaml .Values.global.containerSecuritycontext | nindent 12 }}
           {{- end }}
         {{ end }}
+
+        {{- if eq (include "aggregator.deployMethod" .) "singlepod" }}
+        {{- include "aggregator.containerTemplate" . | nindent 8 }}
+        {{- if .Values.kubecostAggregator.jaeger.enabled }}
+        {{- include "aggregator.jaeger.sidecarContainerTemplate" . | nindent 8 }}
+        {{- end }}
+        {{- end }}
+
+        {{- if eq (include "cloudCost.deployMethod" .) "singlepod" }}
+        {{- include "aggregator.cloudCost.containerTemplate" . | nindent 8 }}
```

Comment:
I appreciate that the indirection here might make this harder to reason about/debug. but damn does it look clean